global.json

```diff
@@ -1,4 +1,7 @@
 {
+  "sdk": {
+    "version": "3.0.100-preview7-012630"
```

Comment:
Do we need to plumb anything for sarcastic to manage this?

Reply:
We currently don't use dependency for for the sdk...

Reply:
Yeah, was wondering if that was the case for everyone (including asp.net) or if we could fix it.

Reply:
I think it makes sense to create an issue to consider flowing the sdk more regularly. In that we can loop darc guys in to figure that out.